tsconfig.json

```diff
@@ -2,7 +2,7 @@
   "compileOnSave": false,
   "compilerOptions": {
     "downlevelIteration": true,
-    "module": "esnext",
+    "module": "es2020",
```

Comment:
What's the impact of this change? Does it only impact internally or also the published package?

Reply:
Not sure. We can try to change it back to esnext and see if it breaks.